test/functional/test_framework/blocktools.py

```diff
@@ -217,3 +219,11 @@ def send_to_witness(use_p2wsh, node, utxo, pubkey, encode_p2sh, amount, sign=Tru
             tx_to_witness = ToHex(tx)
 
     return node.sendrawtransaction(tx_to_witness)
+
+class TestFrameworkScript(unittest.TestCase):
```

Comment:
I think you might have copied this from `script.py`? A better name might be `TestFrameworkBlocktools`